Tasks/DeployAzureResourceGroup/DeployAzureResourceGroup.ps1

```diff
@@ -57,15 +60,23 @@ if($isSwitchAzureModeRequired)
 if( $action -eq "Create Or Update Resource Group" )
 {
     Create-AzureResourceGroupHelper -csmFile $csmFile -csmParametersFile $csmParametersFile -resourceGroupName $resourceGroupName -location $location -overrideParameters $overrideParameters -isSwitchAzureModeRequired $isSwitchAzureModeRequired
+    if(-not [string]::IsNullOrEmpty($outputVariable))
+    {
+        Instantiate-Environment -resourceGroupName $resourceGroupName -outputVariable $outputVariable
+    }
 }
-else
+elseif( $action -eq "SelectRG")
```

Comment:
I was trying to be consistent with DeleteRG :) a shorter name is good as the identifier, what ddo you say?